pkg/apis/sparkscheduler/v1beta2/crd_resource_reservation.go

```diff
@@ -23,7 +23,7 @@ import (
 var v1beta2VersionDefinition = v1.CustomResourceDefinitionVersion{
 	Name:    "v1beta2",
 	Served:  true,
-	Storage: true,
+	Storage: false,
```

Comment:
This should be in a separate PR

Reply:
It was https://github.com/palantir/k8s-spark-scheduler-lib/pull/72, I'll merge develop